person.rb

```diff
@@ -1,15 +1,15 @@
-require './nameable'
+require_relative 'nameable'
 
 class Person < Nameable
```

Comment:
- [Optional] You should have implemented a method used for adding rentals in this class(Person). It seems the previous reviewer missed out on this, and that's why I have marked it **optional** but it's highly required to implement it based on the requirements of this project. The method should look like the piece of code below:

```ruby
     def add_rental(book, date)
           # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
           Rental.new(date, book, self)
     end
```